newsfragments/43486.significant.rst

```diff
@@ -0,0 +1,4 @@
+page_size paramters has been removed from providers/google/ads
```

Comment:
This is provider change not core so newfragment is not relevant.

If the note is segnificant please place it at the top of the google changelog in the provider folder.